### DISCUSSION TITLE
gns3-server on M1 with local docker install

### DISCUSSION THREAD

**Discussion by jhaprins** [GNS3/gns3-gui#3384] [General]
I have a new Apple notebook with the new M1 processor. I can install:
- gns3-gui
- gns3-server
- rosetta
- docker / docker desktop

The docker install is able to run both ARM64 images as x86_64 images. 
It would be fantastic if the gns3-server would be able to use this docker install on the local system so I can run test directly on my laptop. This always worked fine on my Linux laptop. 

Am I missing something, or is it not yet implemented?

Jan Hugo Prins

**Comment by grossmj** [comment]
I am going to have access to a Mac with the new M1 processor, I will run tests to see what is missing.

**Reply by D3M1S22** [reply]
any news on this old topic i'm having the same issue i think.
